smac/facade/epils_facade.py

```diff
@@ -0,0 +1,418 @@
+import logging
```

Comment:
Yes, a lot of code was copied. I don't know whether we should go for a utils.py here or a parent class with some general functions. My intuition would be the latter.

However, I would prefer to do this in another PR and merge this one for now. Ok?